extensions/binding/org.eclipse.smarthome.binding.onewire/src/main/java/org/eclipse/smarthome/binding/onewire/internal/handler/MultisensorThingHandler.java

```diff
@@ -87,7 +89,13 @@ public void initialize() {
         digitalLastRefresh = 0;
 
         // add sensors
-        sensors.add(new DS2438(sensorIds.get(0), this));
+        if (sensorIds.get(0).startsWith("41") || properties.get(PROPERTY_MODELID).equals(OwSensorType.DS1923.name())) {
```

Comment:
see failing test: check if PROPERTY_MODELID exists

Reply:
better do something like
```suggestion
        if (sensorIds.get(0).startsWith("41") || (properties.containsKey(PROPERTY_MODELID) && properties.get(PROPERTY_MODELID).equals(OwSensorType.DS1923.name()))) {
```